src/main/java/org/elasticsearch/index/analysis/SkroutzGreekStemmer.java

```diff
@@ -1067,4 +1102,26 @@ private static CharArraySet loadStopwordSet(InputStream stopwords,
       IOUtils.close(reader);
     }
   }
+
+  /**
+   * Helper function loading an exception Map by its name from the config file.
+   * @param ex_name The exceptions name
+   * @return a Map of "term" -> ['s','t','e','m','m','e','d']
+   * @throws FileNotFoundException
+   * @throws YamlException 
+   */
+  private static Map<String, char[]> loadExceptions(String ex_name)
+          throws FileNotFoundException, YamlException{
+    String fname = SkroutzGreekStemmer.class.getClassLoader()
+            .getResource("stemmer.yml").getFile();
+    YamlReader reader = new YamlReader(new FileReader(fname));
+    Map<String, String> exceptions =(Map) ((Map) reader.read()).get(ex_name);
+    Map<String, char[]> rv = new java.util.HashMap();
+    for (Map.Entry<String, String> entry : exceptions.entrySet()){
```

Comment:
If we're using Java 8 we might consider using lambdas: 

`exceptions.forEach( (k, v) -> rv.put(k.toLowerCase(),
            v.toLowerCase().toCharArray()) );`